### DISCUSSION TITLE
Enabling room clean... am I missing something?

### DISCUSSION THREAD

**Discussion by Kylar514** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#445] [Q&A] [answered]
So I got the card and the map set up, and I am able to do zone clean and pin clean.  But I do not see any room clean option, which is kind of the key feature..  Running roborock S5 Max, and the HA n a docker container.  I see an option for 'generate rooms config' but it is greyed out.  Is this feature available to me?

**Comment by PiotrMachowski** [comment, marked as answer]
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/317

**Reply by Kylar514** [reply]
Yea that was it.  Took me a while to figure that out.  Not only was I missing the "rooms" in the configuration file.  But realizing you edit the code in the card itself took me a while.  New to home assistant in general.    But yes it works.  Thanks!
